src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/NumberFW.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.maven.plugin.internal.generated;
+
+import org.agrona.BitUtil;
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+import org.reaktivity.reaktor.internal.test.types.Flyweight;
+
+public final class NumberFW extends Flyweight
+{
+    private static final int FIELD_OFFSET_VALUE = 0;
+
+    private static final int FIELD_SIZE_VALUE = BitUtil.SIZE_OF_BYTE;
+
+    @Override
+    public int limit()
+    {
+        return offset() + FIELD_SIZE_VALUE;
+    }
+
+    public Number get()
+    {
+        if (-1 == buffer().getByte(offset() + FIELD_OFFSET_VALUE))
```

Comment:
Do we need to support explicitly specifying the value to use for `null`?
What if `-1` is a legal value now used in one of the enum "constructors"?